Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [workspace]
 members = [
-    "contracts/airdrop", 
+    # "contracts/airdrop", 
     "contracts/simple_airdrop", 
-    "contracts/auction", 
-    "contracts/lockdrop", 
+    # "contracts/auction", 
```

Comment:
Cargo.lock is outdated. It needs to run `cargo update`

Reply:
i updated locally but hadn't committed, doing now